src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
```suggestion
  margin-right: 4px;
  background-image: url(images/star.svg);
```

Reply:
Removed the "stars__star--active" modifier and have that built into the "stars__star" as a default background image.

Reply:
please, for better understanding of what you need to do - read requirements and checklist first